recipe/meta.yaml

```diff
@@ -84,7 +84,7 @@ test:
     - if not exist %LIBRARY_PREFIX%\nsight-compute exit 1                      # [win]
     - if exist %LIBRARY_PREFIX%\nsight-compute\{{ version_split }}\lib exit 1  # [win]
     - if not exist %PREFIX%\Scripts\ncu.bat exit 1                             # [win]
-    - ncu --version
+    - ncu --version                                                            # [build_platform == target_platform]
```

Comment:
Yep this seems like the right way to go. Otherwise we are trying to run this on a platform it was not built for, which understandably would have issues